tox.ini

```diff
@@ -1,10 +1,21 @@
 [tox]
-envlist = py26, py27, py32, py33, py34, pypy
+envlist = py26, py27, py32, py33, py34, pypy, gae
 
 [testenv]
+passenv = PYTHONPATH TRAVIS*
 deps= -r{toxinidir}/dev-requirements.txt
 commands=
-   nosetests \
-   []
+    nosetests []
 setenv =
     PYTHONWARNINGS=always::DeprecationWarning
+
+[testenv:gae]
+basepython = python2.7
+setenv=
+    NOSE_IGNORE_CONFIG_FILES = TRUE
```

Comment:
I mean in general, it'd be good to retain the rest of the settings we don't want to override.